cli/create.go

```diff
@@ -45,17 +45,20 @@ func newCreateProjectCmd() *cobra.Command {
 
 // newCreateThemeCmd creates the `verless create theme` command.
 func newCreateThemeCmd() *cobra.Command {
+	var (
+		options core.CreateThemeOptions
+	)
 	createThemeCmd := cobra.Command{
 		Use:   "theme PROJECT NAME",
```

Comment:
not a problem from your PR, but shouldn't that be like this?
```suggestion
		Use:   "theme THEME_NAME",
```